sdks/python/test-suites/portable/py2/build.gradle

```diff
@@ -87,7 +87,7 @@ task crossLanguagePythonJavaDirect {
     ]
     exec {
       executable 'sh'
-      args '-c', ". ${envdir}/bin/activate && cd ${pythonRootDir} && pip install -e .[test] && python -m apache_beam.transforms.external_test ${options.join(' ')}"
+      args '-c', ". ${envdir}/bin/activate && cd ${pythonRootDir} && pip install -e .[test] && python -m apache_beam.transforms.external_java ${options.join(' ')}"
```

Comment:
Please test this also in at least 1 Python 3 suite. If we have to pick one, pick the lowest version (3.5).

Reply:
Done.